src/components/sidenav/sidenav.js

```diff
@@ -299,8 +299,9 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
               ])
               .then(function() {
                 // Perform focus when animations are ALL done...
-                if (scope.isOpen) {
-                  focusEl && focusEl.focus();
+                if (scope.isOpen && focusEl) {
+                  focusEl.focus();
+                  focusEl.triggerHandler('focus');
```

Comment:
The same question I asked myself too. But as I figured out, `focus()` can't be applied to all type of elements. And because the sidenav is a custom angular directive it looks like, they can't be focused through the `focus()` function. That's why I trigger that focus handlers manually, and its working perfect. In my other PR, which already got reviewed by Elad, I'm using the same strategy to get that running (see #5943).

**Quote MDN**
- `The HTMLElement.focus() method sets focus on the specified element, if it can be focused.`